src/Pages/Auth/Register/index.js

```diff
@@ -53,77 +56,128 @@ function Register() {
 
   return (
     <div>
-      <h3>{`${t("int.register")}`}</h3>
+      <h3 className={"u-margin-bottom-large u-text-align-center h6"}>
+        {t("int.register")}
+      </h3>
 
       <form>
-        <label>
-          {`${t("int.firstName")}`}
-          <Input
-            onChange={(value) => setFormData({ ...formData, firstName: value })}
-            type="text"
-            disabled
-            value={formData.firstName}
-          />
-        </label>
-        <label>
-          {`${t("int.lastName")}`}
-          <Input
-            onChange={(value) => setFormData({ ...formData, lastName: value })}
-            type="text"
-            disabled
-            value={formData.lastName}
-          />
-        </label>
-        <label>
-          {`${t("int.dateOfBirth")}`}
-          <Input
-            onChange={(value) => setFormData({ ...formData, birthDate: value })}
-            type="date"
-            disabled
-            value={formData.birthDate}
-          />
-        </label>
-        <label>
-          {`${t("int.profession")}`}
-          <Input
-            onChange={(value) =>
-              setFormData({ ...formData, profession: value })
-            }
-            type="text"
-            disabled
-            value={formData.profession}
-          />
-        </label>
-        <label>
-          {`${t("int.email")}`}
-          <Input
-            type="email"
-            onChange={(value) => setFormData({ ...formData, email: value })}
-            disabled
-            value={formData.email}
-          />
-        </label>
-        <label>
-          {`${t("int.password")}`}
-          <Input
-            onChange={(value) => setFormData({ ...formData, password: value })}
-            type="password"
-            value={formData.password}
-          />
-        </label>
-        <label>
-          {`${t("int.passwordConfirmed")}`}
-          <Input
-            onChange={(value) =>
-              setFormData({ ...formData, passwordConfirmed: value })
-            }
-            type="password"
-            value={formData.passwordConfirmed}
-          />
-        </label>
+        <div className={"u-margin-bottom u-margin-bottom-tiny"}>
+          <label className={cx("c-label c-label--primary c-label--small")}>
+            {`${t("int.firstName")}`}
+
+            <div className={cx("u-margin-bottom", inputWrapper)}>
+              <Input
+                onChange={(value) =>
+                  setFormData({ ...formData, firstName: value })
+                }
```

Comment:
All the disabled inputs shouldn't have an onChange handler. It is useless to them